base/deprecated.jl

```diff
@@ -2093,6 +2093,11 @@ end
     @deprecate chol!(x::Number, uplo) chol(x) false
 end
 
+@deprecate_moved Nullable "Nullables"
+@deprecate_moved NullException "Nullables"
+@deprecate_moved isnull "Nullables"
```

Comment:
I think it's safer to deprecate the specific method for `isnull(::Nullable)`, since `isnull` will be used for `Null`.

Reply:
According to the current plans, `isnull` will be defined in Nulls, which will be either a plain package or a part of the standard library, right? So we need to deprecate the function itself in Base, don't we?